### ISSUE TITLE
IgxIcon text is always announced by Assistive technologies

### ISSUE THREAD

**Issue by ddaribo** [IgniteUI/igniteui-angular#16008]
## Description  

Related to https://github.com/IgniteUI/igniteui-angular/issues/15962
IgxIcon text is always announced by Assistive technologies. It is not always meaningful, and even when it is, it might add a redundancy.

It might be better to add the `aria-hidden=true` to the Icon component by default and leave the option to set it to `false`, if needed. 
Alternatively, we might need to add the `aria-hidden` attribute to many instances of the icon accross the library.

 * igniteui-angular version: any
 * browser: any

## Steps to reproduce  

1. Read various focusable elements from the library containing icons with a screen reader.

## Result  
IgxIcon text is always announced by Assistive technologies

In the below image, first the boolean cell icon (`check`/`close`) is announced instead of the value (`true`/`false`). Additionally, since the column is announced for each cell, `arrow_upward` - the sorting indicator, is appended to the column announcement:

![Image](https://github.com/user-attachments/assets/ce48cf35-e6bc-4d25-abaa-cda937d5afb9)

![Image](https://github.com/user-attachments/assets/5bfb5982-1907-4b8d-8987-82e8aa78d65e)

## Expected result  
Do not announce the icon text by default.

## Attachments  
Attach a sample if available, and screenshots, if applicable.
